### ISSUE TITLE
Community Ladder for Non-code contributors

### ISSUE THREAD

**Issue by oleg-nenashev** [keptn/community#84]
Currently https://github.com/keptn/community/blob/master/COMMUNITY_MEMBERSHIP.md is heavily focused on Code contributions. This is not correct, because other kinds of contributions are as valuable as Code ones, e.g. documentation, design, advocacy&outreach, reviews, etc. In addition to #82, I believe we should update the community chatter to allow onboarding non-code contributors and promoting them up to the official leading roles in the community.

Basically it implies splitting the "Maintainer" and "Community Leader" roles which are currently equivalents in the community

## Maintainer sponsors

- @AloisReitbauer 
- @grabnerandi 
- @johannes-b 
- @thisthat 
- @jetzlstorfer
